alua.c

```diff
@@ -319,6 +323,8 @@ tcmu_get_tgt_port_grp(struct tcmu_device *dev, const char *name)
 	free(orig_str_val);
 	return group;
```

Comment:
Oh yeah, I merged as is assuming it was for future proofing the code.

If there are other strings being returned let me know the kernel version and I can fix that.